package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@hmcts/div-test-harness",
+  "version": "1.0.0",
+  "description": "Module used for functional tests to create a case and progress it to a certain state so that other tests can build upon it",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hmcts/div-test-harness.git"
```

Comment:
Is it git+url? here

Reply:
Yes. This was added by npm init when it recognised that github is the where this repo is.
Do you have something else you'd like me to put here instead @dtaylor7 ?